### PR TITLE
Include a warning icon before blocks that return errors

### PR DESCRIPTION
**Pull request by jacebrowning**
TODO:

- [x] Add tooltip
- [x] Apply to every page
- [ ] Apply to every language?

### CODE REVIEW

**Comment by brondsem** [comment]
This looks like a good step forward to me.  Would it be possible to include title/hover text on the icon?  That might be nice if someone forgets what it means.

**Comment by jacebrowning** [comment]
@brondsem Yeah, I can add that. Should this change be applied to every language before merging?

File: en/python_introduction/README.md [hunk 1]
```diff
@@ -498,7 +502,7 @@ You can give Python as many numbers to compare as you want, and it will give you
 
 Have you heard of the expression "comparing apples to oranges"? Let's try the Python equivalent:
 
-{% filename %}command-line{% endfilename %}
+{% filename %}{{ warning_icon }} ccommand-line{% endfilename %}
```
Comment (brondsem) [inline en/python_introduction/README.md]: Typo "ccommand"

**Comment by jacebrowning** [comment]
I fixed the typo.

Initially I included this icon on a few of the later sections, but I took it out because:

- the icon was only needed in one place (less to keep updated)
- the students might be comfortable with errors at that point

**Comment by jacebrowning** [comment]
@rsip22 @ekohl @aniav What do you think about merging this? We have a workshop coming up in just over a week and I think this will really help people! 😄

**Comment by RachellCalhoun** [comment]
@jacebrowning do you have a gitbook example up that you can share so we can see these changes?

**Comment by jacebrowning** [comment]
This is what it looks like:

<a href="https://cl.ly/2y0L3O3l0F0b" target="_blank"></a>

**Pull request merged**